### ISSUE TITLE
test: fix skipped tests after react v18 update

### ISSUE THREAD

**Issue by escapedcat** [cstenglein/raspiblitz-web#234]
Some tests are `skip`ed at the moment, because they didn't work anymore after the update.  
Those should be fixed.

- [x] ConfirmSendModal
- [x] SwitchTxType
- [x] UnlockModal
- [ ] SendOnChain

**Comment by escapedcat** [comment]
Two `SendOnChain` test still have issues with the `amount`-input.\
`validates amount is bigger than zero`-test might be related to this, but also not sure:
https://user-images.githubusercontent.com/1016218/174426204-147abd09-8a1f-4f54-a81d-1b25201e95f1.mov

**Comment by cstenglein** [comment]
Thank you for the investigation!

The test failure is indeed correct. I found the issue and fixed that in https://github.com/cstenglein/raspiblitz-web/pull/358

The value is usually a number when validation runs, but will be displayed as a string (so the separator gets added).

When submitting, the value is a string and that is an issue with the validation. As a fix I used the amount from the state, but that was lagging behind the onChange input value, which caused the issue you filmed. :)

Only 1 test missing now :)
